pyvista/utilities/parametric_objects.py

```diff
@@ -1,4 +1,9 @@
-"""Module managing parametric objects."""
+"""Module managing parametric objects.
+
+These objects represent surfaces that are parametrised by a set of independent
```

Comment:
It doesn't look like this makes it into the documentation.  It needs to be added to https://github.com/pyvista/pyvista/blob/main/doc/api/utilities/parametric.rst